### PR TITLE
Customize badge radius

### PR DESCRIPTION
**Pull request by kf99916**
As title

### CODE REVIEW

**Review by Minitour — CHANGES_REQUESTED** [review]
Can remove non-swift file changes?

**Comment by kf99916** [comment]
Fixed.

File: Sources/EasyNotificationBadge/EasyNotificationBadge.swift [hunk 1]
```diff
@@ -229,6 +233,7 @@ public struct BadgeAppearance {
     public var duration: TimeInterval
     public var distanceFromCenterY: CGFloat
     public var distanceFromCenterX: CGFloat
+    public var radius: CGFloat
```
Comment (Minitour) [inline Sources/EasyNotificationBadge/EasyNotificationBadge.swift]: This should be optional.
```suggestion
    public var radius: CGFloat?
```

File: Sources/EasyNotificationBadge/EasyNotificationBadge.swift [hunk 2]
```diff
@@ -83,10 +83,14 @@ extension UIView {
...
         // calculate width and height with minimum height and width of 20
-        let height = max(18, (badgeSize?.height ?? 0.0) + 5.0)
-        let width = max(height, (badgeSize?.width ?? 0.0) + 10.0)
+        let badgeSize = badgeLabel?.frame.size
+        var height = max(18, (badgeSize?.height ?? 0.0) + 5.0)
+        var width = max(height, (badgeSize?.width ?? 0.0) + 10.0)
+        if appearance.radius > 0 {
+            height = appearance.radius
+            width = appearance.radius
+        }
```
Comment (Minitour) [inline Sources/EasyNotificationBadge/EasyNotificationBadge.swift]:
```suggestion
        if let radius = appearance.radius {
            height = radius
            width = radius
        }
```

File: Sources/EasyNotificationBadge/EasyNotificationBadge.swift [hunk 3]
```diff
@@ -242,5 +247,6 @@ public struct BadgeAppearance {
         allowShadow = false
         distanceFromCenterY = 0
         distanceFromCenterX = 0
+        radius = 0
```
Comment (Minitour) [inline Sources/EasyNotificationBadge/EasyNotificationBadge.swift]: Remove default value

**Comment by kf99916** [comment]
Fixed and thank you four your suggestion.

**Review by Minitour — APPROVED** [review]

**Comment by Minitour** [comment]
I will release this later on today. @kf99916 Thank for your contribution.

**Pull request merged**